### DISCUSSION TITLE
Please Help! Unable to run meshlab

### DISCUSSION THREAD

**Discussion by nestorio** [cnr-isti-vclab/meshlab#1344] [Q&A]
Hello! Thank you very much for this great tool. I am new to meshlab and I would like to run the meshlab code locally. I followed the following steps as mentioned in the git repo:

```
git clone --recursive https://github.com/cnr-isti-vclab/meshlab
mkdir meshlab/build
cd meshlab/build
cmake ..
make
```
I succeeded to complete the steps despite some warnings. However, I do not know what to do next to run the build. I was expecting to find an executable meshlab file after the build but there is no such file. I then decided to try opening it with Qt using the following steps as mentioned in the documentation:

You can also use [QtCreator](https://www.qt.io/product) to build MeshLab:

    Install QtCreator and Qt 5.15;
    Open CMakeLists.txt;
    Configure and build MeshLab.

Unfortunately, it still did not work. I get the following error

21:16:41: Starting /home/user/test-meshlab/build-meshlab-Desktop_Qt_5_15_2_GCC_64bit-Debug/src/distrib/Char16Test...
21:16:41: The process failed to start. Either the invoked program "/home/user/test-meshlab/build-meshlab-Desktop_Qt_5_15_2_GCC_64bit-Debug/src/distrib/Char16Test" is missing, or you may have insufficient permissions to invoke the program.

I am stuck at this point and do no know how to proceed. I would really like to run the source code so as to implement the project am working on.

Thank you for your time and kind reply

**Comment by HungNgoCT** [comment]
I have the same problem. Did you solve it?

Can anyone help me?

Many thanks
